numpy/random/_generator.pyx

```diff
@@ -380,6 +380,23 @@ cdef class Generator:
         out : ndarray or scalar
             Drawn samples from the parameterized exponential distribution.
 
+        Examples
+        --------
+        A real world example: Assume a company has 10000 customer support 
+        agents and each agent spends on average 4 minutes with a 
+        customer.
+
+        >>> n = 10000
+        >>> z = np.random.default_rng().exponential(scale=4, size=n)
```

Comment:
It seems you are calling it `d` below, so the test is failing.  But maybe best is to give it a meaningful name, one thought would be `call_durations = ...`?

However, I do wonder if an exponential is a good model for this (I would expect there to be a clear peak in typical call durations, which it does not have: there are no 2 second long calls), maybe we can use something that is known to follow an exponential distribution?

Reply:
@seberg Gotcha, I'll change the name to something more specific. What if I changed the problem context to be the number of minutes before a customer walks into a branch location for a store?

Reply:
@lzha97, maybe the time between calls should be right, I think?  But I generally any poissant process should do (in physics radioactive decay, photons hitting a sensor, etc.)?  We could even just not add a close example.

Anyway, would be nice to push it forward, and the only thing is to fix the test and best make sure the example doesn't seem a strange fit with an exponential distribution!